update-versions.sh

```diff
@@ -21,50 +21,57 @@ set -ex
 HEAD=$(git rev-parse HEAD)
 VERSION=$1
 
+# ordered in a topological fashion starting from the workspace root Cargo.toml
+# this isn't always necessary, but it's nice to use a mostly-consistent ordering
 CARGO_TOMLS_TO_BUMP=(
     ./Cargo.toml
-    ./pgx/Cargo.toml
+    ./pgx-pg-config/Cargo.toml
```

Comment:
:( Sorry about that.

Reply:
It's okay, really! I shouldn't have missed it in review.